mycroft/skills/intent_service.py

```diff
@@ -39,7 +36,11 @@ def _get_message_lang(message):
     Returns:
         The language code from the message or the default language.
     """
+    message = message or dig_for_message()
+    # TODO read active locale from LF instead
     default_lang = Configuration.get().get('lang', 'en-us')
+    if not message:
+        return default_lang
```

Comment:
Consider logging a warning here (might be duplicated in `dig_for_message`, needs testing)

Reply:
no warning, this would be a valid use case, lang is not mandatory in message